mmaction/models/detectors/base.py

```diff
@@ -0,0 +1,137 @@
+from abc import ABCMeta, abstractmethod
+from collections import OrderedDict
+
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+
+
+class BaseDetector(nn.Module, metaclass=ABCMeta):
```

Comment:
What's the difference between this version and the mmdet version

Reply:
Some functions in mmdet ver `BaseDetector` are not implemented now (aug_test, etc. ). Besides, some interface is not aligned with MMCV. The refactor may take some time, maybe in the next version we can use the one in mmdet as the base class.